app/src/main/java/fr/iut_rodez/pathpilot_android_client/util/Popup.java

```diff
@@ -0,0 +1,35 @@
+package fr.iut_rodez.pathpilot_android_client.util;
+
+import android.app.AlertDialog;
+import android.content.Context;
+import android.widget.Toast;
+
+public class Popup {
```

Comment:
Missing Javadoc comment to describe what the class is for

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/38/commits/eb823270691dc0c912b801466992d2ce62dfca4e